From: "Barak A. Pearlmutter" <barak+git@cs.nuim.ie>
Date: Mon, 16 Sep 2013 22:52:11 +0100
Subject: vobcopy.h xMNTINFO

Fix the autotool-ification to work on kfreebsd, per
http://bugs.debian.org/697016 by accepting the patch therein.

Thanks to: Petr Salinger <Petr.Salinger@seznam.cz>
---
 vobcopy.h | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/vobcopy.h b/vobcopy.h
index 33cdf52..1da5ce4 100644
--- a/vobcopy.h
+++ b/vobcopy.h
@@ -124,16 +124,8 @@ typedef enum  { FALSE=0, TRUE=1 }  bool;
 
 #ifdef HAVE_GETMNTINFO
 #define USE_GETMNTINFO
-#endif
-
-#ifdef USE_GETMNTINFO
-#ifdef USE_STATFS_FOR_DEV
 #define GETMNTINFO_USES_STATFS
 #endif
-#ifdef USE_STATVFS_FOR_DEV
-#define GETMNTINFO_USES_STATVFS
-#endif
-#endif
 
 #include <dvdread/dvd_reader.h>
 
